backend/src/main/java/net/wodie/backend/service/BtoService.java

```diff
@@ -24,8 +28,49 @@ public BtoItem updateBtoItem(BtoItem updatedBtoItem) {
     }
 
     public String getBtoItemStatusById(String id) {
-        return  btoRepository.findById(id)
-                .orElseThrow( () -> new NoSuchElementException("BtoItem not found with id: " + id))
+        return btoRepository.findById(id)
+                .orElseThrow(() -> new NoSuchElementException(BTO_ITEM_NOT_FOUND + id))
                 .getStatus();
     }
+
+    @SuppressWarnings("java:S3776")
```

Comment:
A comment why this warning is suppressed would be good here - for me .. you know. ;)